Livecheckables/sslscan.rb

```diff
@@ -0,0 +1,6 @@
+class Sslscan
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)-rbsec$/i)
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)(?:-rbsec)?$/i)
```

It would probably be good to make the `-rbsec` optional, in case they drop it from tags anytime in the future.